### PR TITLE
docs: Modify README.md about wait_limit option of batch-run command

### PR DESCRIPTION
**Pull request by gkzz**
### Motivation
I'd like to share with you that you can also make a use of `wait_limit` option, when using the `wait` command to wait for the `./magicpod-api-client batch-run` command to finish.

### What I changed
I added an example of the `wait_limit` option.

Ref. https://github.com/Magic-Pod/magicpod-api-client/blob/beb27eebd35222cfd55c779fba324bf505dc11d1/magicpod-api-client.go#L43-L45

### What I checked
Checked manually if comparing the output of `./magicpod-api-client batch-run --help` command.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -75,6 +75,21 @@ wait $PID1
 wait $PID2
 ```
 
+> **Note**: The default of Wait limit in seconds is 300 seconds.
+
+You can also specify the wait limit in seconds. If execution time of your [BatchRun/CrossBatchRun](https://magic-pod.com/api/v1.0/doc/) is expected to exceed the initial value of 300 seconds, it would be better to specify as follows.
```
Comment (nkns165) [inline README.md]: I'm afraid `magic-pod.com` is our old domain. Could you replace it with `app.magicpod.com`?
We already know the redirection from https://magic-pod.com/api/v1.0/doc/ to https://app.magicpod.com/api/v1.0/doc/ is not working while it works on other URLs, and consider it should be fixed on our side in the future.
Sorry for the inconvenience.

Comment (nkns165) [reply README.md]: It seems the default value 300 seconds is for another command `get-screenshots`.
The waiting time for `batch-run` is `test count x 10 minutes` as you mentioned.
Could you please replace the description?

**Review by nkns165 — CHANGES_REQUESTED** [review]
Thank you for your contribution again, I have two requests.

File: README.md (hunk 1 above)
Comment (gkzz) [reply README.md]: Thank you for your kindness!
I applied your feedback, https://github.com/Magic-Pod/magicpod-api-client/pull/6/commits/7759c22789590dc4420f8f6c1cdd8c71152577c7

I'll rebase it, if everything is ok :)

Comment (nkns165) [reply README.md]: Thank you for your fix!
Sorry, I have told you a wrong comment in the previous pull request.
We usually do `squash and merge`, not `rebase and merge`. By `squash and merge`, all changes are gathered into one commit on the main branch. So you don't need to rebase the commits in the pull request.

**Review by nkns165 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (gkzz) [reply README.md]: I got it.
I've completed it, https://github.com/Magic-Pod/magicpod-api-client/pull/6/commits/18205dab4dbf51d1d7c9f48925b3423123e48aa7

**Pull request merged**